src/components/data-management/SamplesTable.jsx

```diff
@@ -48,6 +50,18 @@ const exampleDatasets = [
   },
 ];
 
+// If a sample exists in projects but not in the samples file,
+// there is an error uploading the sample. We should show an error for that sample.
+const sampleCreateErrorTemplate = {
+  ...sampleTemplate,
+  name: 'UPLOAD ERROR: Please reupload sample',
```

Comment:
now that you've made the experiments table add the sample name, you can display it directly rather and remove this `name`

Reply:
removed